src/event_imp.rs

```diff
@@ -1004,7 +1015,7 @@ impl Interests {
     /// [`Poll`]: struct.Poll.html
     #[inline]
     pub fn readable() -> Interests {
-        Interests(READABLE)
+        Interests(NonZeroUsize::new(READABLE).unwrap())
```

Comment:
It think it's ok to use `new_unchecked` here, this would allow it to be a `const fn` in the future

Reply:
Is it worth to add the `unsafe` blocks in this context?

Reply:
I think so as the only possible invalid behaviour is passing 0 to `new_unchecked`, which is easy to check for. I haven't checked this but I think removing the `unwrap` should be good for performance and as I said it would allow the function to be constant.